cloudburst/server/executor/server.py

```diff
@@ -211,7 +210,10 @@ def executor(ip, mgmt_ip, schedulers, thread_id):
                     ((len(received_triggers[trkey]) == len(schedule.triggers)) \
                     or (fref.type == MULTIEXEC))):
 
-                triggers = list(received_triggers[key].values())
+                triggers = list(received_triggers[trkey].values())
+
+                if fname not in function_cache:
+                    logging.error('%s not in function cache', fname)
```

Comment:
If this happens, shouldn't we return an error instead of just continuing on?

Reply:
This happens when the management node dereplicates the function but the scheduler node hasn't been notified promptly so it keeps scheduling a dag onto an executor that already evicted the function. Yes we should signal an error, any idea what exactly we should do in this case?

Reply:
fixed